enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/actions/NamespaceProcessor.java

```diff
@@ -131,9 +120,8 @@ private List<VariantItem> getSortedVariants(String prefix) {
             String ns = it.next();
             Library library = supportedLibraries.get(ns);
             if (prefix.equals(library.getDefaultPrefix())) {
-                if (jsfSupport != null && jsfSupport.isJsf22Plus()) {
-                    ns = library.getNamespace();
-                }
+                ns = jsfVersion.getNamespaceUri(prefix);
```

Comment:
The original code used the namespace from the library - should this not be possible anymore?

Reply:
With the new `JsfVersion` enum this is not necessary anymore as the namespace information is part of that enum.